backend/api/services/postServices.py

```diff
@@ -164,8 +164,12 @@ def getPaginatedPosts(request, res, author_id, pageNum, pageSize):
   # return: formatted json response
   #################################################
   def formatJSONpost(request, post, author_id, post_id = ''):
-    urlParts= 'http://' + request.get_host() + '/author/' + author_id + '/posts/' + post_id
     formedJsonRes = {}
+    if post['author_id']:
```

Comment:
@PENGYUXIONG  will get rid of this stuff, will merge ur refactors in first

Reply:
cool, my change is in now